app/controllers/hub/clients/organizations_controller.rb

```diff
@@ -36,13 +36,13 @@ def load_vita_partners
         @vita_partners = VitaPartner.accessible_by(current_ability)
       end
 
-      def authorize_vita_partner
-        raise CanCan::AccessDenied unless @vita_partners.find_by(id: parsed_vita_partner_id).present?
-      end
-
-      def parsed_vita_partner_id
-        return nil unless client_params[:vita_partners].present?
-        JSON.parse(client_params[:vita_partners]).pluck("id")
+      def load_vita_partner
+        id = JSON.parse(client_params[:vita_partners]).pluck("id").first
```

Comment:
Non-blocker: If `JSON.parse()` can raise exceptions, it can be nice to catch them. The best way to handle them would probably be to treat it the same as when the vita_partner can't be found, i.e., the `head :forbidden` section.